package.json

```diff
@@ -40,19 +40,33 @@
     ]
   },
   "devDependencies": {
+    "@babel/core": "^7.15.0",
```

Comment:
Holy POOP, that's a lot of new dev-deps, each one to be lovingly cared for and updated every eight minutes when Remnovate Bot gets its knickers in a twist.

Is there no other way? No-one maintains a metapackage that includes the up-to-date versions of all these? For that matter, `@babel/eslint-parser` doesn't even have them as deps itself?

Reply:
(I note that https://babeljs.io/docs/en/babel-plugin-proposal-numeric-separator says "**NOTE:** This plugin is included in `@babel/preset-env`".)

Reply:
I agree, too many dev-deps. I anticipated they would be managed solely by `stripes-webpack`, but lint wouldn't run for me unless I specified them here, too. At the time, I grimaced and moved on, blinded by the glory of a clean lint run, but it's worth another look. Thanks for encouraging me to make this better.

Reply:
I must've had a crufty `node_modules` directory because, in fact, all those new deps _are_ managed by `stripes-webpack` and lint runs just fine without specifying them here. Thanks again; this is much cleaner now.